docs/installation/index.md

```diff
@@ -82,43 +72,46 @@ yum-config-manager --disable "*"
 yum-config-manager --enable rhel-6-server-rpms epel
 yum-config-manager --enable rhel-6-server-optional-rpms
 ```
+</div>
 
-**RHEL7**
-
-```bash
-yum -y  --disablerepo="*" --enablerepo=rhel-7-server-rpms install yum-utils wget
-yum-config-manager --disable "*"
-yum-config-manager --enable rhel-7-server-rpms epel
-yum-config-manager --enable rhel-7-server-optional-rpms
-yum-config-manager --enable rhel-7-server-extras-rpms
-```
-
-### Enteprise Linux
-
-For all variations of enterprise linux (CentOS and RHEL) the following steps need to be taken:
-
-**EL6**
+<div id="el6" markdown="1">
+**All Enterprise Linux 6 Variants**
```

Comment:
I'd just drop the 'All' here and contemplate dropping the title all together. With the selector displaying exactly my choice, this ends up seeming repetitive.